src/main/java/dev/cdevents/CDEventTypes.java

```diff
@@ -5,97 +5,713 @@
+            final String taskRunData)
+    {
+        CloudEvent ceToSend =
+                buildCloudEvent(eventType, taskRunData)
+                        .withExtension("id", id)
+                        .withExtension("source", source)
+                        .withExtension("taskname", taskName)
+                        .withExtension("pipelinerun", pipelineRun)
+                        .withExtension("url", url)
+                        .build();
+        return ceToSend;
+    }
+
+    /**
+     * Creates a continuous delivery Task run finished events
+     * using {@link CloudEventBuilder}
+     * and returns {@link CloudEvent} object.
+     *
+     * @param eventType
+     * @param id
+     * @param source
+     * @param taskName
+     * @param pipelineRun
+     * @param url
+     * @param outcome
+     * @param errors
+     * @return the cloudEvent object with continuous delivery
+     * task run finished event extensions
+     */
+
+
+    public static CloudEvent createTaskRunFinishedEvent(
+            final String eventType, final String id,
+            final URI source, final String taskName,
+            final String pipelineRun, final URI url,
+            final CDEventConstants.Outcome outcome, final String errors,
+            final String taskRunData)
+    {
+        CloudEvent ceToSend =
+                buildCloudEvent(eventType, taskRunData)
+                        .withExtension("id", id)
+                        .withExtension("source", source)
+                        .withExtension("taskname", taskName)
+                        .withExtension("pipelinerun", pipelineRun)
+                        .withExtension("url", url)
+                        .withExtension("outcome", outcome.getOutcome())
+                        .withExtension("errors", errors)
+                        .build();
+        return ceToSend;
+    }
+
+
+    /**
+     * Creates a continuous delivery repository created event
+     * using {@link CloudEventBuilder}
+     * and returns {@link CloudEvent} object.
+     *
+     * @param eventType
+     * @param id
+     * @param source
+     * @param name
+     * @param owner
+     * @param url
+     * @param view
+     * @param data
+     * @return the cloudEvent object with continuous delivery
+     * repository created event extensions
+     */
+
+    public static CloudEvent createRepositoryCreatedEvent(
+            final String eventType, final String id,
+            final URI source, final String name,
+            final String owner, final URI url,
+            final URI view, final String data) {
+        CloudEvent ceToSend =
+                buildCloudEvent(eventType, data)
+                        .withExtension("id", id)
+                        .withExtension("source",source)
+                        .withExtension("name", name)
+                        .withExtension("owner",owner)
+                        .withExtension("url", url)
+                        .withExtension("view", view)

```

Comment:
update this with the ("viewUrl", viewUrl), this field is `viewUrl` as per the spec